docs/DeveloperGuide.md

```diff
@@ -467,6 +467,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 6. **Make `add` and `edit` commands accept blacklisted client status:** The current implementation throws an error of invalid command format for the parameter `cs/blacklisted`. Users can work around this by using the blacklist command implemented. In the future, we plan to remove the restriction on blacklisted client status from both `AddCommandParser` and `EditCommandParser`.<br>
 7. **Make rejection of blacklisted client status for add and edit command more specific:** This is an alternative to point 6 above. The current error message thrown is `"Invalid command format!` which does not accurately explain to the user that the issue lies with the blacklisted parameter. We plan to make the error message mention that `cs/blacklisted cannot be used for this command`. **It is important to note that this enhancement will be deemed redundant if point 6 is implemented, where `cs/blacklisted` is indeed accepted.**<br>
 8. **Reject email inputs without a period between alphanumeric characters for the domain portion:** Currently, an input like `e/john@example` is accepted, however, this should not be the case as a valid domain format would require at least 1 period for the top level domain (ie.john@example.com). We plan to adjust the regex for the domain portion of the email from `"(" + DOMAIN_PART_REGEX + "\\.)*"` to `"(" + DOMAIN_PART_REGEX + "\\.)+"` to reject such an input.<br>
+9. **Make `INDEX` 0 and negative `INDEX` error messages more specific:** Currently, when 0 or negative numbers are passed for the `INDEX` parameter, the error message thrown is `Invalid command format!`. In the future, we will change this error message to `The person index provided is invalid`.
```

Comment:
```suggestion
9. **Make `INDEX` 0 and negative `INDEX` error messages more specific:** Currently, when 0 or negative numbers are passed for the `INDEX` parameter, the error message thrown is `Invalid command format!`. In the future, we will change this error message to `The person INDEX provided is invalid`.
```

Reply:
Our current index message is not caps so i standardised